Week-1/Session-02/Assignments/Intermediate/Poem.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Poem Webpage</title>
+</head>
+
+<body>
+    <p>
+        I'm Nobody! Who are you?<br>
+        Are you - Nobody - Too?<br>
+        Then there's a pair of us!<br>
+        Don't tell! They'd banish us - you know!
+    </p>
+
+    <p>
+        How dreary - to be - somebody!<br>
+        how public - like a Frog -<br>
+        To tell one's name - the livelong June -<br>
+        To an Admiring Bog!
+    </p>
+
+    <p>>
+        <cite>By Emily Dickinson(1891)</cite>
```

Comment:
```suggestion
        <cite>By Emily Dickinson (1891)</cite>
```

Reply:
Sure, I will update this suggested changes.